nodejs/eks/index.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016-2018, Pulumi Corporation.
+// Copyright 2016-2019, Pulumi Corporation.
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
```

Comment:
I would have expected to need a new export in this file to export out the `InputTags` type.  Possibly not a huge deal - but makes it hard for users to mention it's name.

Reply:
Done.